### DISCUSSION TITLE
主应用vue2报We're sorry but system doesn't work properly without JavaScript enabled. Please enable it to continue.

### DISCUSSION THREAD

**Discussion by LQS5858** [Tencent/wujie#291] [Q&A]
主应用vue2，安装wujie-vue2加载子应用vue2报We're sorry but system doesn't work properly without JavaScript enabled. Please enable it to continue.

**Comment by unicycle-master** [comment]
一样的问题，怎么解决

**Comment by unicycle-master** [comment]
自己写了一个静态的页面，正常显示，要是加上js代码，js不运行。不关vue的问题，是不支持运行js
